go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/nicklaw5/go-respond
+
+go 1.12
```

Comment:
Is there any real need to include this file if we don't have any dependencies?

Reply:
Yes, otherwise i could not use it outside my gopath. The package names did not resolve right in the tests.

Reply:
I don't believe the absence of a go.mod file would break your tests. Can you provide an example of the issue you're running into?

Reply:
Hi Nick, i did not say it breaks the build generally. It Just cant find the right package if you are outside of the gopath.

Why is this such a big thing? Go Modules are a Feature added in Go 1.12. They do not give any disadvantages and you could just ignore them if working not with go 1.12.

Reply:
You may to pull in your dependencies before running test. For example

```bash
go get ./...
go test ./...
```

Reply:
When i delete the go.mod file and try to test this project outside from $GOPATH or $GOROOT, it will give me following error output:

```
PS C:\Users\Alex\Desktop\go-respond> go test ./
# _/C_/Users/Alex/Desktop/go-respond
error_test.go:8:2: cannot find package "github.com/nicklaw5/go-respond" in any of:
        C:\Go\src\github.com\nicklaw5\go-respond (from $GOROOT)
        D:\Go\src\github.com\nicklaw5\go-respond (from $GOPATH)
FAIL    _/C_/Users/Alex/Desktop/go-respond [setup failed]
PS C:\Users\Alex\Desktop\go-respond>
```

Adding this file would remove the need of placing this project inside your $GOPATH or $GOROOT, even if you don't have any dependencies. You declare a package-name which would otherwise not be found.

Reply:
I see what you mean. That can be fixed. Let me fix it and then we rebase your branch onto master.

Reply:
Ok that's been fixed on the `master` branch. You'll need to rebase this branch:

```
git rebase master
```

Then apply the changes in https://github.com/nicklaw5/go-respond/commit/4ab9d9b07919521752a190ee107f699ce3020b34.

You should then be able to run your tests without any issue.

Reply:
I merged your changes, removed the `go.mod` file and fixed some more importing issues.